README.md

```diff
@@ -1,2 +1,40 @@
-# AtlasPack
-Combined cortical/subcortical atlases for xcp-d and qsiprep
+# Combining cortical and subcortical atlases for use in XCPD and QSIPrep
+
+There are many popular brain atlases, but they're all in different versions of MNI space. They also cover 
+different structures. We want to get them all into one MNI version correctly - using the templateflow 
+framework - so they can be used to make structural and functional connectivity matrices.
+
+## Things to worry about
+
+We ultimately want square connectivity matrices with exactly the same regions represented in the rows/columns to come out of xcpd 
+and qsiprep. Here are some of the problems we will run into:
+
+ 1. Some regions are so small that when they're mapped to a low-res BOLD/DWI volume they will disappear. Should we allow this? 
+    In some cases they are from dividing a region into subcomponents. We could recombine them.
+ 2. Regions may overlap each other from different atlases. Which should have precedence?
```

Comment:
If I remember correctly, qsiprep needs everything to be in the same atlas file, right? Because xcpd can just concatenate time series files if necessary, I think.

If there are overlapping regions, we could use a 4D atlas file, and basically pretend it's a probabilistic segmentation.

Reply:
Mrtrix, DSI Studio, etc need a single 3D volume :(